src/collective/cover/testing.py

```diff
@@ -97,6 +97,11 @@ def setUpZope(self, app, configurationContext):
             self.loadZCML(package=Products.PloneFormGen)
             z2.installProduct(app, 'Products.PloneFormGen')
 
+        # Install plone.app.imaging to make image scales work
+        z2.installProduct(app, 'plone.app.imaging')
```

Comment:
@alecpm do we depend on `plone.app.imaging`? it would be better to add it as a dependency on `setup.py` and register it in `configure.zcml`